src/external/rawspeed/data/cameras.xml

```diff
@@ -5734,6 +5734,40 @@
 		<Crop x="0" y="0" width="-74" height="0"/>
 		<Sensor black="15" white="3971"/>
 	</Camera>
+	<Camera make="Panasonic" model="DMC-LX15">
+		<ID make="Panasonic" model="DMC-LX15">Panasonic DMC-LX15</ID>
```

Comment:
I believe http://www.imaging-resource.com/PRODS/panasonic-lx10/panasonic-lx10A.HTM says that LX9, LX10 and LX15 are the same camera. And that page has DMC-LX10 sample.